src/common/tests/btree.c

```diff
@@ -997,13 +899,100 @@ static struct option btr_ops[] = {
 	{ NULL,		0,			NULL,	0	},
 };
 
+static void
+ts_group(void ** state){
+	int	opt = 0;
+	void	**st = NULL;
+	while ((opt = getopt_long(test_group_stop-test_group_start+1,
+				  test_group_args+test_group_start,
+				  "tmC:Deocqu:d:r:f:i:b:p:",
+				  btr_ops,
+				  NULL)) != -1) {
+		tst_fn_val.optval = optarg;
+		tst_fn_val.input = true;
+
+		switch (opt) {
+		case 'C':
+			ik_btr_open_create(st);
+			break;
+		case 'D':
+			tst_fn_val.input = true;
+			ik_btr_close_destroy(st);
+			break;
+		case 'o':
+			tst_fn_val.input = false;
+			tst_fn_val.optval = NULL;
+			ik_btr_open_create(st);
+			break;
+		case 'c':
+			tst_fn_val.input = false;
+			ik_btr_close_destroy(st);
+			break;
+		case 'e':
+			ik_btr_drain(st);
+			break;
+		case 'q':
+			ik_btr_query(st);
+			break;
+		case 'u':
+			tst_fn_val.opc = BTR_OPC_UPDATE;
+			ik_btr_kv_operate(st);
+			break;
+		case 'f':
+			tst_fn_val.opc = BTR_OPC_LOOKUP;
+			ik_btr_kv_operate(st);
+			break;
+		case 'd':
+			tst_fn_val.opc = BTR_OPC_DELETE;
+			ik_btr_kv_operate(st);
+			break;
+		case 'r':
+			tst_fn_val.opc = BTR_OPC_DELETE_RETAIN;
+			ik_btr_kv_operate(st);
+			break;
+		case 'i':
+			ik_btr_iterate(st);
+			break;
+		case 'b':
+			ik_btr_batch_oper(st);
+			break;
+		case 'p':
+			ik_btr_perf(st);
+			break;
+		default:
+			D_PRINT("Unsupported command %c\n", opt);
+		case 'm':
+		case 't':
+			/* handled previously */
+			break;
+		}
+	}
+}
+
+static int
+run_cmd_line_test(char* test_name, char** args, int start_idx, int stop_idx)
```

Comment:
(style)  "foo* bar" should be "foo *bar"
(style)  "foo** bar" should be "foo **bar"